apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx

```diff
@@ -131,7 +143,25 @@ export function ConnectIntegrationForm({
             />
             <StyledText>{isActive ? 'Active' : 'Disabled'}</StyledText>
           </ActiveWrapper>
-        </RowDiv>
+          {createModel && provider?.channel === ChannelTypeEnum.EMAIL && (
+            <CheckIntegrationWrapper>
+              <Controller
+                control={control}
+                name="check"
+                render={({ field }) => (
+                  <Switch
+                    checked={isCheckIntegration}
+                    data-test-id="is_check_integration_id"
+                    {...field}
+                    onChange={handlerCheckIntegrationChange}
+                  />
+                )}
+              />
+              <StyledText>Send a test email to verify the credentials?</StyledText>
+            </CheckIntegrationWrapper>
+          )}
+        </Stack>
```

Comment:
Currently, the option to test connection will be only enabled for create new integration and Email channel

Reply:
Could we make it work also for "update" ?

Reply:
Done ;)